README.md

```diff
@@ -131,16 +134,18 @@ That command will generate:
  * a jar file for the annotations
  * a jar file for the annotations processor
 
+The full build may take an hour or more, to complete.
+
 ### Other Commands
 
  * `./gradlew tasks` will show all the available tasks
  * `./gradlew javadoc` will generate the Javadocs
  * `./gradlew monkeyExamples` will run the monkey tests on all the examples
  * `./gradlew installRealmJava` will install the Realm library and plugin to mavenLocal()
- * `./gradlew clean -PdontCleanJniFiles` will remove all generated files except for JNI related files. This saves recompilation time a lot.
- * `./gradlew connectedUnitTests -PbuildTargetABIs=$(adb shell getprop ro.product.cpu.abi)` will build JNI files only for the ABI which corresponds to the connected device.
+ * `./gradlew clean -PdontCleanJniFiles` will remove all generated files except for JNI related files. This reduces recompilation time a lot.
+ * `./gradlew connectedUnitTests -PbuildTargetABIs=$(adb shell getprop ro.product.cpu.abi)` will build JNI files only for the ABI which corresponds to the connected device.  These tests require a running Object Server (see below)
```

Comment:
I think can do that the same way I separated `connectedUnitTests` and `connectedBenchmarks`. It requires that the tests are placed in another package name though, otherwise we cannot configure the testrunner to ignore them. That is outside the scope of this PR though.